app/scripts/modules/google/instance/gceMultiInstanceTask.transformer.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular
+  .module('spinnaker.google.instance.multiInstance.task.transformer', [])
+  .factory('gceMultiInstanceTaskTransformer', function () {
+
+    // custom transformers for specific tasks, e.g. "rebootInstances" needs an empty "interestingHealthProviders" array
+    let transformers = {
+      rebootInstances: (job) => {
+        job.interestingHealthProviders = [];
```

Comment:
interestingHealthProviders => interestingHealthProviderNames here and in the above comment.